pyproject.toml

```diff
@@ -22,6 +22,7 @@ spacy = {version = "^3", optional = true}
 nltk = {version = "^3", optional = true}
 transformers = {version = "^4", optional = true}
 beautifulsoup4 = {version = "^4", optional = true}
+pytest-cov = "^4.0.0"
```

Comment:
can we add this to the `tool.poetry.group.test.dependencies` group instead? shouldn't be needed for the built package right?